.circleci/config.yml

```diff
@@ -21,9 +21,33 @@ jobs:
             pip install -t lib/ -r requirements.txt
             pip install nose nosegae nose-exclude
             nosetests --with-gae --gae-lib-root=`gcloud info --format="value(installation.sdk_root)"`/platform/google_appengine --exclude-dir=lib
-          
+
+  tag:
+    docker:
+      - image: alpine/git
+
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "ab:2d:12:de:d1:ac:36:0b:6a:02:92:3f:2c:49:98:c9"
```

Comment:
is this safe to be putting into source?  (i'm no ssh expert :( )

Reply:
im just learning this now but the fingerprint is a hash of the public key so it's okay to publicize